notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb

```diff
@@ -0,0 +1,1289 @@
+{
```

Comment:
<p>Use the Vertex SDK if possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>The current Python SDK's create index does not support set index update method...</p>